source/codegen/metadata/nixnet/functions.py

```diff
@@ -141,16 +140,17 @@
             {
                 'direction': 'in',
                 'name': 'frameBuffer',
-                'size': {
-                    'mechanism': 'len',
-                    'value': 'numberOfBytesForFrames'
-                },
-                'type': 'void[]'
+                'type': 'u8',
```

Comment:
note: Changed type from `void[]` to `u8` because `validate_metadata` expects size mechanism to be present on input arrays. This was easier way to get around that.